### ISSUE TITLE
Circular bounding layout

### ISSUE THREAD

**Issue by vfutia** [Jawnnypoo/PhysicsLayout#21]
Maybe I'm missing it, but there does not appear to be a way to create a PhysicsFrameLayout (or other layout) that's boundary is a circle.  I'm looking to put circle views inside a circular bounding layout.  Is there a way to do this?

**Comment by Jawnnypoo** [comment]
Yes, this is not currently supported, but it is an interesting idea. You could achieve something similar by flagging the layout to not have bounds and to manually create circular bounds yourself as an "invisible" view within the layout

**Comment by vfutia** [comment]
Thanks for getting back to me!  How would I go about creating a circular bounds within the layout?  Is this something I would have to do manually using Box2D or is it something that could be achieved with your library?  Sorry, I'm not trying to get you to code my solution or anything... just wanted some clarification.  If you could point me in the right direction it would be appreciated.

**Comment by Dhameliya** [comment]
Hi, i am using PhysicsLayout  for bounding view or animate view, but i want circle boundary instead of rectangle, is there any solution for achieve boundary with circle?

**Comment by Ein-xinlong** [comment]
> @ghost 你好，我正在使用 PhysicsLayout 作为边界视图或动画视图，但我想要圆形边界而不是矩形，有没有解决方案可以实现圆形边界？

I also have this requirement here. Do you have a solution? Thank you

**Comment by Ein-xinlong** [comment]
> Thanks for getting back to me! How would I go about creating a circular bounds within the layout? Is this something I would have to do manually using Box2D or is it something that could be achieved with your library? Sorry, I'm not trying to get you to code my solution or anything... just wanted some clarification. If you could point me in the right direction it would be appreciated.

I also have this requirement here. Do you have a solution? Thank you
